components/kyma-metrics-collector/pkg/process/metrics.go

```diff
@@ -48,8 +48,29 @@ var (
 		},
 		[]string{shootNameLabel, instanceIdLabel, runtimeIdLabel, subAccountLabel, globalAccountLabel},
 	)
+	kebTotalClusters = promauto.NewGaugeVec(
```

Comment:
Can we use counter here? Seems like we are going to reset the metric anyway.

Reply:
true, done!